_data/docs.yml

```
@@ -183,6 +193,16 @@ libs:
       legacy: 1
       stable: 1
       nightly: 1
+  libcugraphops:
+    name: libcugraphops
+    path: libcugraphops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
```

```suggestion
      legacy: 0
```

`cugraph-ops` only recently started publishing documentation, so they don't have legacy (`23.02`) docs